src/webrequest.js

```diff
@@ -660,23 +662,33 @@ chrome.runtime.onMessage.addListener(function (request, sender, sendResponse) {
     }
 
   } else if (request.superCookieReport) {
-    if (Utils.hasSuperCookie(request.superCookieReport)){
+    if (badger.utils.hasSuperCookie(request.superCookieReport)){
       recordSuperCookie(sender, request.superCookieReport);
     }
   } else if (request.checkEnabledAndThirdParty) {
     var pageHost = window.extractHostFromURL(sender.url);
-    sendResponse(Utils.isPrivacyBadgerEnabled(tabHost) && window.isThirdParty(pageHost, tabHost));
+    sendResponse(badger.utils.isPrivacyBadgerEnabled(tabHost) && window.isThirdParty(pageHost, tabHost));
   } else if (request.checkSocialWidgetReplacementEnabled) {
-    sendResponse(Utils.isPrivacyBadgerEnabled(tabHost) && Utils.isSocialWidgetReplacementEnabled());
+    sendResponse(badger.utils.isPrivacyBadgerEnabled(tabHost) && badger.utils.isSocialWidgetReplacementEnabled());
   }
+}
 
-});
+/*************** Event Listeners *********************/
+function startListeners() {
```

Comment:
Thanks for catching all these! I couldn't find an auto linting to find and fix them.